src/form.js

```diff
@@ -414,9 +414,13 @@ define([
                 return node.getValue();
             });
         },
-        updateError: function (errObj, options) {
+        /**
+         * Add parsing error to the form
+         *
+         * NOTE these errors are displayed on form load only.
+         */
+        updateError: function (errObj) {
```

Comment:
The old-style error is still used for form parse [errors](https://github.com/dimagi/Vellum/blob/3b17849889117e21264739ee95779fe8af6c29af/src/parser.js#L99-L100) and [warnings](https://github.com/dimagi/Vellum/blob/3b17849889117e21264739ee95779fe8af6c29af/src/parser.js#L171-L183), which are not associated with a mug.